repos/dir.props

```diff
@@ -53,6 +53,7 @@
     <EnvironmentVariables Include="DotNetCoreSdkDir=$(DotNetCliToolDir)" />
     <EnvironmentVariables Include="DotNetBuildToolsDir=$(ProjectDir)Tools/" />
     <EnvironmentVariables Include="StabilizePackageVersion=$(UseStableVersions)" Condition="'$(UseStableVersions)' != ''" />
+    <EnvironmentVariables Include="PB_IsStable=$(UseStableVersions)" Condition="'$(UseStableVersions)' != ''" />
```

Comment:
This is from https://github.com/dotnet/source-build/commit/cdf7249ccb14bf1b1da499f0c283b78129556036, we do need it for proper versions.